### PR TITLE
fix(www): make GitHub icon above plugin readme clearer

### PR DESCRIPTION
**Pull request by kbrgl**
It was hard to find and isn't very clear.

## Description
Adds 'View plugin on GitHub' next to the GitHub icon, and introduces some spacing to make it easier to find. It wasn't very clear earlier where it goes (view source on GitHub could mean either page source or plugin source.)

Just a minor something I've clashed with a few times 😄 

screen shot 2019-02-27 at 2 59 46 am

### CODE REVIEW

File: www/src/components/package-readme.js [hunk 1]
```diff
@@ -40,9 +42,8 @@ const PackageReadMe = props => {
           }}
           href={githubUrl}
           aria-label={`${packageName} source`}
-          title={`View source on GitHub`}
         >
-          <GithubIcon focusable="false" style={{ verticalAlign: `text-top` }} />
+          <GithubIcon focusable="false" style={{ verticalAlign: `text-top` }} />&nbsp;&nbsp;<span>View plugin on GitHub</span>
```
Comment (m-allanson) [inline www/src/components/package-readme.js]: Can you remove the `&nbsp;&nbsp;` and instead add the spacing with CSS?

File: www/src/components/package-readme.js [hunk 2]
```diff
@@ -40,9 +42,8 @@ const PackageReadMe = props => {
           }}
           href={githubUrl}
           aria-label={`${packageName} source`}
```
Comment (m-allanson) [inline www/src/components/package-readme.js]: Maybe remove this `aria-label` and replace it with an [`aria-labelledby` attribute](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute) that links to the text `<span>View plugin on GitHub</span>`? 

cc @marcysutton is that best practice here?

Comment (wardpeet) [reply www/src/components/package-readme.js]: I'm pretty sure they do the same thing but it's better to not repeat yourself so 👍 for `aria-labelledby`

File: www/src/components/package-readme.js (hunk 1 above)
Comment (kbrgl) [reply www/src/components/package-readme.js]: Yep, no problem!

File: www/src/components/package-readme.js (hunk 2 above)
Comment (kbrgl) [reply www/src/components/package-readme.js]: Sure 👍

**Comment by kbrgl** [comment]
@m-allanson the text doesn't fit at 320px, so I've added a media query to hide it at widths of 385px and below. Here's a look at the text at just under 385px:

screenshot_2019-02-27 gatsby-plugin-sharp

And just over:

screenshot_2019-02-27 gatsby-plugin-sharp 1

I've fixed the other issues.

**Comment by fk** [comment]
Hey @kbrgl, thanks so much for this PR — I completely agree with the changes, 💯 to both.

Wondering if we should slightly adjust the "See starters that use this" to "See starters using this"?

In regards to the small screen issues: Looking at #11753 incoming, I think we won't get around breaking/wraping items into rows instead of hiding labels. We also want to unfiy link styles 😅, but IMO that's a thing for another PR.

**Comment by kbrgl** [comment]
Hi, @fk!

Here's how that'll look:

screenshot_2019-02-27 gatsby-plugin-react-helmet

**Comment by fk** [comment]
@kbrgl #12253

File: www/src/components/package-readme.js [hunk 3]
```diff
@@ -27,22 +27,35 @@ const PackageReadMe = props => {
...
               "&:hover": {
                 background: `none`,
                 color: colors.gatsby,
               },
             },
           }}
           href={githubUrl}
-          aria-label={`${packageName} source`}
-          title={`View source on GitHub`}
+          aria-labelledby="github-link-label"
```
Comment (marcysutton) [inline www/src/components/package-readme.js]: For future reference, `aria-labelledby` isn't needed on a link if it contains text content.

**Pull request merged**